### PR TITLE
Feature/packing list complete

### PR DESCRIPTION
**Pull request by miriam-ertl**
[## User Story 12 – Packing list Complete#16](https://github.com/miriam-ertl/my-trips/issues/16)

- [ ] Acceptance criteria
- [ ]  Each item in the packing list has one checkbox
- [ ]  This checkbox can be clicked when the item is packed
- [ ]  Clicking on a checkbox the item will be crossed out and it is marked as checked and
- [ ]  Moved below the packing list in a "done"-area
- [ ]  Clicking on the checkbox again the item is unmarked und switches to the unmarked items

### CODE REVIEW

File: pages/trips/[id]/index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import ConfirmDelete from "@/components/ConfirmDelete";
 import Image from "next/image";
+import ItemList from "@/components/ItemList";
```
Comment (e-rmkt) [inline pages/trips/[id]/index.js:3]: I think this import is not needed here, correct? As ItemList is used in PackingList? :)

File: pages/trips/[id]/index.js [hunk 2]
```diff
@@ -14,10 +15,11 @@ export default function DetailsPage() {
     isLoading,
     mutate,
   } = useSWR(id ? `/api/trips/${id}` : null);
+
   if (!trip || isLoading) {
     return "... is loading";
   }
-
+  console.log(trip);
```
Comment (e-rmkt) [inline pages/trips/[id]/index.js:22]: If I am not mistaken, then this console.log is left over and can be removed? :)

File: pages/trips/[id]/index.js [hunk 3]
```diff
@@ -71,6 +73,34 @@ export default function DetailsPage() {
...
+    const completeItems = trip.packingList.map((completeItem) =>
+      completeItem._id === _id
+        ? { ...completeItem, checked: !completeItem.checked }
+        : { ...completeItem }
+    );
+    const updatedTrip = {
+      ...trip,
+      packingList: [...completeItems],
+    };
+    console.log(updatedTrip);
```
Comment (e-rmkt) [inline pages/trips/[id]/index.js:86]: Another leftover log maybe? :)

**Review by e-rmkt — COMMENTED** [review]
LGTM, very nice code, fun to read through it! Just see my comments regarding some leftiver logs or imports! :D

File: pages/trips/[id]/index.js (hunk 1 above)
Comment (miriam-ertl) [reply pages/trips/[id]/index.js]: @e-rmkt thank you for your very nice comment. And also for your review of cause :-D.
So unneeded imports and also console.logs are removed.

**Pull request merged**